deploy/olm-catalog/integreatly-operator/integreatly-operator-1.19.0/integreatly-operator.v1.19.0.clusterserviceversion.yaml

```diff
@@ -258,14 +253,16 @@ spec:
                     fieldRef:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
-                  value: integreatly-operator
+                  value: rhmi-operator
                 - name: USE_CLUSTER_STORAGE
                   value: "true"
-                image: quay.io/integreatly/integreatly-operator:v1.18.0
+                - name: ALERTING_EMAIL_ADDRESS
+                  value: "integreatly-notifications@redhat.com"
```

Comment:
@philbrookes Looks good.

To confirm, his would only be set in addon environments from stage, is that correct? This won't be set in any dev environments? If that's the case, we're all good

Reply:
ah, I see then I should remove it from here.